### ISSUE TITLE
Some SSL sites still hang LWP [rt.cpan.org #82153]

### ISSUE THREAD

**Issue by oalders** [libwww-perl/libwww-perl#172]
Migrated from [rt.cpan.org#82153](https://rt.cpan.org/Ticket/Display.html?id=82153) (status was 'new')

Requestors:
* am@jobindex.dk

From am@jobindex.dk on 2012-12-21 10:18:42:
```
This program will currently hang:

     perl -e 'use LWP::Simple; print get("https://www.geocaching.com");'


It is apparently fixed by this patch, but I have a feeling that it might 
break something else?

--- a/perl-5.16.2/lib/site_perl/5.16.2/Net/HTTP/Methods.pm
+++ b/perl-5.16.2/lib/site_perl/5.16.2/Net/HTTP/Methods.pm
@@ -233,7 +233,6 @@ sub my_read {
             return length($_[0]);
         }
         else {
-           die "read timeout" unless $self->can_read;
             return $self->sysread($_[0], $len);
         }
      }



Best regards,
Anders.

-- 
Anders Melchiorsen, Programmer, am@jobindex.dk
http://www.jobindex.dk/

```

From am@jobindex.dk on 2013-01-07 15:43:14:
```
Probable duplicate of https://rt.cpan.org/Ticket/Dis play.html?id=81237 
<https://rt.cpan.org/Ticket/Display.html?id=81237>

DOY's fix works for me.

```

**Comment by genio** [comment]
Fixed as of 6.06

**Comment by genio** [comment]
https://rt.cpan.org/Public/Bug/Display.html?id=81237
